relayer-cosmos/src/cosmos/all_for_one/chain.rs

```diff
@@ -32,6 +34,7 @@ pub trait AfoCosmosChainContext<Counterparty>:
     WriteAcknowledgementEvent = WriteAcknowledgement,
     ConsensusState = ConsensusState,
     ChainStatus = ChainStatus,
+    Telemetry = OfaTelemetryWrapper<CosmosTelemetry>,
```

Comment:
There is no need for a concrete type for `Telemetry`, if there is no compile error and the external user has no need to know about the concrete type. In general the AllForOne traits are used for finding the smallest closure dependencies required for specific uses.

Reply:
Thank you ! I removed it in the latest commit.